src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -1501,6 +1500,10 @@ public async Task Http2_PendingSend_SendsReset()
                     Assert.NotNull(frame); // We should get Rst before closing connection.
                     Assert.Equal(0, (int)(frame.Flags & FrameFlags.EndStream));
                  } while (frame.Type != FrameType.RstStream);
+
+                 frame = await connection.ReadFrameAsync(TimeSpan.FromMilliseconds(TestHelper.PassingTestTimeoutMilliseconds)).ConfigureAwait(false);
+                 // Make sure we do not get any frames after getting RST.
```

Comment:
nit: for consistency with the 'Assert.NotNull(frame)' above, the comment should be on the same line:

```c#
Assert.Null(frame); // Make sure we do not get any frames after getting RST.
```